modules/fedimint-mint-client/src/lib.rs

```diff
@@ -11,6 +11,8 @@
 
 // Backup and restore logic
 pub mod backup;
+/// Modularized Cli for sending and receiving out-of-band ecash
+mod cli;
```

Comment:
We probably want to put it behind a feature flag, like in other modules. But can be done separately.